test/index.js

```diff
@@ -6,27 +6,29 @@ var locales = ['en-US', 'fr-FR', 'de-FR'];
 var path = require('path');
 var appRoot = path.resolve(__dirname);
 
-var writer = function (root, cb) {
-	return function (out) {
+var writer = function(root, cb) {
+	return function(out) {
 		var deleted;
-		for (var locale in out) {
+		for(var locale in out) {
```

Comment:
In general, style changes should not be mixed with functional changes. In a PR, as separate commits, or preferably, as separate PRs, since the goal of a style change and who it serves is different than the goal of a functional change and who it serves.